.github/workflows/tests_lkcpu_python.yml

```diff
@@ -34,7 +34,7 @@ on:
 env:
   TF_VERSION: 2.10.0
   TORCH_VERSION: 1.11.0+cpu
-  COVERAGE_FLAGS: "--cov=pennylane_lightning --cov-report=term-missing --no-flaky-report -p no:warnings --tb=native"
+  COVERAGE_FLAGS: "--cov=pennylane_lightning --cov-report=term-missing -p no:warnings --tb=native"
```

Comment:
Is this something that should be temporary?

Reply:
Yes, It is something temporal. I will remove it

Reply:
Done!